### ISSUE TITLE
Where is the database

### ISSUE THREAD

**Issue by jasperliinla2020** [DaniloNovakovic/apartment-reservation#1]
database or SQL scripts?

**Comment by DaniloNovakovic** [comment]
@jasperliinla2020 For SQL (database for commands), in the default configuration `appsettings.json` file, the database is configured to be generated automatically using the [_Code-First_](https://www.entityframeworktutorial.net/code-first/what-is-code-first.aspx) approach on the _localdb_ instance which you can access through Visual Studio. 

If you wish to change the location of your database then you need to change the connection string in the previously mentioned file (tables & data should be automatically created/seeded when you first start application with your new connection string).

The same goes for MongoDB (database for queries) - the default location is also localhost (requires MongoDB to be installed on your local machine by default) which you can access by either a command line or, preferably, through GUI software like [MongoDB Compass](https://www.mongodb.com/products/compass)

In summary: Just set up the configuration strings in `appsettings.json` and databases should be generated automatically when you start up the application, otherwise, they will be located on your local machine by default 😊
